raiden/tests/integration/transfer/test_refund_invalid.py

```diff
@@ -22,23 +23,25 @@ def test_receive_secrethashtransfer_unknown(raiden_network, token_addresses):
     token_network_address = views.get_token_network_address_by_token_address(
         views.state_from_app(app0), app0.raiden.default_registry.address, token_address
     )
+    assert token_network_address
 
     other_key = HOP1_KEY
     other_signer = LocalSigner(other_key)
     canonical_identifier = factories.make_canonical_identifier(
         token_network_address=token_network_address
     )
 
-    amount = 10
+    amount = TokenAmount(10)
+    locksroot = Locksroot(UNIT_SECRETHASH)
```

Comment:
Should we have something like `UNIT_LOCKROOT`  here?

Reply:
Isn't it a wrong value on purpose, here?

Reply:
Ok, then it might be nicer to use `make_20bytes` to show that this has nothing to do with the secrethash.